### ISSUE TITLE
Prediction with variable batch size

### ISSUE THREAD

**Issue by ilaripih** [Russell91/TensorBox#127]
Hi,

Is there a straightforward way to use an input of n images when doing predictions instead of the default 1? So the shape of `x_in` would be `[None, height, width, 3]` instead of `[height, width, 3]`.

**Comment by Russell91** [comment]
Yes, there is actually a batch_size parameter in the json file which should allow you to do multiple prediction.

**Comment by ilaripih** [comment]
Thanks, I got it to work after making some changes to the evaluate.py code and the `add_rectangles` function (it currently handles only the first image).

**Comment by GeorgiAngelov** [comment]
@ilaripih, do you mind sharing your changes to add_rectangles to handle the n images situation? I am currently exploring the exact same thing.
